### PR TITLE
 Make an alternative method while cloning

### PR DESCRIPTION
**Pull request by JohnNiang**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

https://github.com/apache/skywalking/issues/4331
___
### New feature or improvement

Make an alternative method while cloning and reformat some code blocks

### CODE REVIEW

**Comment by JohnNiang** [comment]
Preiview: <https://github.com/JohnNiang/skywalking/blob/chore/doc/docs/en/guides/How-to-build.md>

File: docs/en/guides/How-to-build.md [hunk 1]
```diff
@@ -17,11 +17,19 @@ If you need to execute build behind the proxy, edit the *.mvn/jvm.config* and pu
...
+    ```bash
+    git clone --recurse-submodules https://github.com/apache/skywalking.git
+    cd skywalking/
+    
+    OR
+    
+    git clone https://github.com/apache/skywalking.git
+    cd skywalking/
+    git submodule init
+    git submodule update
```
Comment (wu-sheng) [inline docs/en/guides/How-to-build.md]: Your step is not right, because branch change could relate to submodule change.

Comment (JohnNiang) [reply docs/en/guides/How-to-build.md]: Got it.

Comment (JohnNiang) [reply docs/en/guides/How-to-build.md]: Maybe we could specify the branch name / tag name in the after `git clone command`.

**Pull request merged**